doc/source/flexibility/innodb_fts_improvements.rst

```diff
@@ -54,3 +53,38 @@ System Variables
 
 When enabled, this variable will instruct |InnoDB| Full Text Search
 parser to ignore the stopword list when building/updating an FTS index.
+
+.. _punctuation_marks:
+
+Punctuation Marks in Full-Text Search
+=====================================
+
+By default, full text search is unable to find words with various punctuation
+characters in boolean search mode, although those characters are
+indexed with ngram parser. A new variable :variable:`ft_query_extra_word_chars`
+was introduced in |Percona Server| :rn:`5.7.21-20` to solve this issue.
+
+When it's enabled, all the non-whitespace symbols are considered to be
+word symbols by FTS query parser, except for the boolean search syntax
```

Comment:
link to upstream variable specifying the syntax symbols

Reply:
done